WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdViewModel.swift

```diff
@@ -154,6 +160,10 @@ final class BlazeEditAdViewModel: ObservableObject {
             guard let image = await onAddImage(source) else {
                 return imageState = previousState
             }
+            guard image.image.size.width >= minImageSize.width || image.image.size.height >= minImageSize.height else {
```

Comment:
I'm not sure if checking the size is enough. I have an image of size 1722*501 but it still got rejected when creating a new campaign. In order to test this, you can open any image in Preview app > Tools > Adjust Size > then set either of the dimensions to 500 or 501. I haven't checked with slightly larger sizes, it'd be great if we can confirm the logic with backend folks.

Reply:
I have updated the expected size as 600*600. But we are still talking with the backend team as the campaign creation fails for image size 1722*601. p1712848101999269/1712831053.533739-slack-C069RG07W6S

Meanwhile, since we started showing the exact error reason to user I think we can merge this PR for now and update the image size validate again.